lib/composite.js

```diff
@@ -147,6 +148,14 @@ function composite (images) {
         throw is.invalidParameterError('gravity', 'valid gravity', image.gravity);
       }
     }
+    if (is.defined(image.premultiplied)) {
+      if (is.bool(image.premultiplied)) {
+        composite.premultiplied = image.premultiplied;
+      } else {
+        throw is.invalidParameterError('premultiplied', 'boolean', image.premultiplied);
```

Comment:
Please can you add a unit test to cover the line that throws an Error should `image.premultiplied` be defined but not a boolean.

Reply:
Hopefully this is consistent with the test for "invalid tile"